tests/PHPStan/Reflection/PhpDefect/PhpDefectClassReflectionExtensionTest.php

```diff
@@ -347,12 +347,12 @@ public function dataDomEntityProperties(): array
 					'nodeName' => 'string',
 					'nodeValue' => 'string',
 					'nodeType' => 'int',
-					'parentNode' => \DOMNode::class,
+					'parentNode' => 'DOMNode|null',
```

Comment:
did you intenionally drop the `\` root-namespace prefix?

Reply:
Hi Marcus. 
Nothing has changed with the namespace - <code>\DOMNode::class === 'DOMNode'</code>.

The commit just added <code>|null</code> to the type definition. It could be written as <code>\DOMNode::class.'|null'</code> but that would not be in keeping with the other union types defined in this class.

Reply:
it could also be `'\DOMNode|null'`

Reply:
String with backslash wouldn’t work.

Reply:
Canonical class names will never have a leading slash - PHP will accept them in functions like class_exists but this is only for convenience. Phpstan needs the true canonical class name to match against <code>ReflectionClass->getName()</code>

Reply:
@betterthanclay You're exactly right.